browserdebuggertools/sockethandler.py

```diff
@@ -103,11 +105,13 @@ def _recv(self):
         return message
 
     def _get_websocket_url(self, port):
-        targets = requests.get(
+        response = requests.get(
             "http://localhost:{}/json".format(port), timeout=self.CONN_TIMEOUT
-        ).json()
-        logging.debug(targets)
-        tabs = [target for target in targets if target["type"] == "page"]
+        )
+        if not response.ok:
+            raise DevToolsException("Cannot connect to browser on port %s" % port)
```

Comment:
This will hide the original exception, including for instance the status code you got. Is that what you want?